src/components/Input/Input.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { ErrorMessage, useField } from 'formik';
+import {
+  DivStyles, LabelStyles, InputStyles, PasswordControl,
+} from './Input.styles';
+import { WarningTextStyles } from '../Text/WarningText';
+import eye from '../../assets/icons/eye.png';
+import eyeSlash from '../../assets/icons/eye-slash.png';
+
+export const Input = ({
+  label, isEyeVisible, fontSize, ...props
+}) => {
+  const [field] = useField(props);
+  const [visible, setVisible] = useState(false);
+
+  const changeVisibility = () => {
+    setVisible(!visible);
+  };
+
+  return (
+    <DivStyles>
```

Comment:
DivStyles is not suitable, div is not informative

Reply:
renamed